projects/angular-auth-oidc-client/src/lib/config/config.service.ts

```diff
@@ -112,23 +112,30 @@ export class ConfigurationService {
     const usedConfig = this.prepareConfig(passedConfig);
     this.saveConfig(usedConfig);
 
-    const alreadyExistingAuthWellKnownEndpoints = this.storagePersistenceService.read('authWellKnownEndPoints', usedConfig);
+    const configWithAuthWellKnown = this.addWellKnownEndpoint(usedConfig);
+
+    this.publicEventsService.fireEvent<OpenIdConfiguration>(EventTypes.ConfigLoaded, configWithAuthWellKnown);
+
+    return of(usedConfig);
+  }
+
+  private addWellKnownEndpoint(configuration: OpenIdConfiguration): OpenIdConfiguration {
```

Comment:
This name can be improved? It is not just add I think

Reply:
Improved the name